ui/app/templates/components/auth-jwt.hbs

```diff
@@ -44,13 +44,14 @@
     </div>
   {{/unless}}
   <div data-test-yield-content>
+    {{! Custom mount path input renders here }}
     {{yield}}
   </div>
 
   {{#if this.isOIDC}}
     <Hds::Button
-      @text={{concat "Sign in with " (or this.role.providerName "OIDC Provider")}}
-      @icon={{if @disabled "loading" this.role.providerIcon}}
+      @text={{concat "Sign in with " (or this.fetchedRole.providerName "OIDC Provider")}}
+      @icon={{if @disabled "loading" this.fetchedRole.providerIcon}}
```

Comment:
I don't think we can use optional chaining in the hbs template, but wondering how this would handle the situation where `this.fetchedRole` is `null`. Maybe it's worth assigning it `providerName` and `providerIcon` to variables in the class that can handle a `null` value?

Reply:
Good callout! HBS let's us get away with a lot 🙃 , for example, I could set `role` to `{ foo: bar }` and even though `providerIcon` doesn't even exist on `role` the template still renders just fine: 
<img width="610" alt="Screenshot 2025-04-01 at 4 59 26 PM" src="https://github.com/user-attachments/assets/1516b5de-2d31-41de-8d76-f719744b08e0" />
<img width="908" alt="Screenshot 2025-04-01 at 5 00 10 PM" src="https://github.com/user-attachments/assets/3164ab6d-2bba-4658-a568-c6cbf052dab6" />

This component is going to get a complete rewrite in the auth form, so I've intentionally left everything as close to the original code as possible. But this is a good suggestion and I'll keep it in mind for the new oidc component!